### DISCUSSION TITLE
Next.js + Excalibur.js

### DISCUSSION THREAD

**Discussion by the-man-with-a-golden-mind** [excaliburjs/Excalibur#2664] [General]
Hello, 
I am struggling to use Excalibur.js in my Next.js project. I've tried to run Excalibur breakout example. Canvas is loading but its empty. Do you have any example of Excalibur and Next.js? 
The site with Excalibur is rendered on client side not backend. I am loading excelibur and adding actors in useEffect to be sure that it would be on client side. 

Thanks, 

MM

**Comment by the-man-with-a-golden-mind** [comment]
I've found similar discussion with solution: [https://github.com/excaliburjs/Excalibur/discussions/1886](url)

**Comment by kimikoi123** [comment]
There is a next-excalibur template here https://codesandbox.io/s/next-excalibur-5b18x. You can also apply the concept if you want to create the latest version of nextJs
